<launch>
    <node pkg="icreate_navigation" type="waypoint_navigation" name="waypoint_navigation" output="screen">
        <param name="move_base_topic" value="/move_base" type="string" />
        <param name="lift_file_path" value="/waypoint/build4_f20.csv" type="string" />
        <param name="base_frame_id" value="/map" type="string" />
        <param name="package_name" value="icreate_navigation" type="string" />
    </node>
</launch>